### ISSUE TITLE
there should be an error for one changepoint with one data point

### ISSUE THREAD

**Issue by tdhock** [diego-urgell/BinSeg#18]
hi @diego-urgell in your edge case test file I see
```r
ans <- BinSeg::BinSegModel(100, "BS", "mean_norm", 1)
```
There is one data point (100) with nCpts=1 (fourth argument), so there should be an error. It is not possible to compute any changepoints if there is only one data point.

**Comment by diego-urgell** [comment]
Right, it makes sense since it is not possible to "partition" the dataset if there is only one data point. Thanks!
